kubespawner/reflector.py

```diff
@@ -150,16 +152,18 @@ def _list_and_update(self):
 
         Overwrites all current resource info.
         """
-        initial_resources = getattr(self.api, self.list_method_name)(
+        method = partial(getattr(self.api, self.list_method_name), _preload_content=False)
+        initial_resources = method(
             self.namespace,
             label_selector=self.label_selector,
             field_selector=self.field_selector,
             _request_timeout=self.request_timeout,
         )
```

Comment:
Is it correct that introducing partial accomplished the same thing as just adding _preload_content=False to the list of arguments? If so, is there a motivation to using partial over just adding the argument to the list?

```suggestion
        initial_resources = getattr(self.api, self.list_method_name)(
            self.namespace,
            label_selector=self.label_selector,
            field_selector=self.field_selector,
            _request_timeout=self.request_timeout,
            _preload_content=False,
        )
```

Reply:
There was no reason. I just copied what I did in `_watch_and_update` (where partial application is required).